### PR TITLE
add Interaction design draft

### PR DESCRIPTION
**Pull request by spiderbiggen**
sorry no latex

As a developer
I want to have good interaction design

### CODE REVIEW

File: documents/Interaction Design/analysis.tex [hunk 1]
```diff
@@ -0,0 +1,24 @@
+\section{Persona}
+This section describes the software architecture views, it explains the subsystem decomposition, the hardware/software mapping, persistent data management and the concurrency.
```
Comment (MichielvdBerg) [inline documents/Interaction Design/analysis.tex:2]: This introduction is not about software arhcitecture views right?

**Comment by jzoon** [comment]
This document is not complete. I believe there are quite some things that can be added, like details on the interaction between the agent and actual people. Also, in the Persona section the agent isn't even mentioned in the little stories: shouldn't this be added? Or maybe add a person that adds indicators for it's role, but doesn't play himself and lets an agent play for him. Now you just describe persons that play the Tygron game.

Would be nice if you can improve this document.

**Comment by spiderbiggen** [comment]
I should indeed add a persona for someone who can't be there himself but let's the agent work for him

**Comment by tvleest** [comment]
Michiel and me will start improving this document. 
I'll close the pull request for now.

**Pull request closed without merging**